lib/coercion.ex

```diff
@@ -45,7 +45,12 @@ defmodule Coercion do
       {:ok, "true"}
       iex> coerce(10.5, :string)
       {:ok, "10.5"}
-
+      iex> :bellow
```

Comment:
Looks good to merge, but did you mean "bellow" or "below", which are two completely different words. Just wanted to make sure this was what you intended.

Reply:
`bellow` is accurate